kuma/wiki/templates/wiki/email/edited.ltxt

```diff
@@ -32,4 +32,14 @@ Edit Article:
 Article History:
 {% endtrans %}
  {{ history_url|absolutify }}
-{% if watch %}{{ unsubscribe_text(watch) }}{% endif %}{% endautoescape %}
+--
+{% if watch %}
+  {% set title = watch.content_object.title %}
+    {% if watch.event_type == 'wiki edit document in tree' %}
+      {{ _('You are subscribed to edits on: {title} and all its sub-articles.')|f(title=title) }}
+    {% else %}
+      {{ _('You are subscribed to edits on: {title}.')|f(title=title) }}
+    {% endif %}
```

Comment:
Since this is an email, do you want to use jinja2 whitespace eating tags? Or do you want those strings indented?